### PR TITLE
Enhancement in the accuracy of the logs regarding the capacity, usage, and threshold of secondary storages

### PR DESCRIPTION
**Pull request by lucas-a-martins**
### Description

The `capacity` and `usage` values from secondary storage shown in the logs lack accuracy due to the absence of decimal places, creating cases where the value is rounded down even if it is close to the higher number (e.g., 49.90 GB being shown as 49GB).

This pull request addresses this issue by adding two decimal places to `capacity` and `usage` values. Additionally, in order to standardize, the `threshold` value, which was more accurate than necessary, was reduced to only two decimal places.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

### How Has This Been Tested?

I tested by registering a template and verifying the logs.

Before changes:

```
2024-05-03T19:33:48,616 DEBUG [c.c.s.StatsCollector] (secstorage-1:[ctx-391b7254]) (logid:bcdb0d20) Verifying image storage [1]. Capacity: total=[58 GB], used=[4 GB], threshold=[89.99999761581421%].
```

After changes:

```
2024-05-03T18:12:37,198 DEBUG [c.c.s.StatsCollector] (secstorage-1:[ctx-e10bb7c0]) (logid:f8f066d7) Verifying image storage [1]. Capacity: total=[58.80 GB], used=[4.10 GB], threshold=[90.00%].
```

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Review by gpordeus — APPROVED** [review]
@lucas-a-martins 

LGTM, although perhaps 'threshold' could be put into an intermediate string so we can use the {} notation, but it's more of a nitpick.

File: server/src/main/java/com/cloud/server/StatsCollector.java [hunk 1]
```diff
@@ -1823,16 +1823,16 @@ public boolean imageStoreHasEnoughCapacity(DataStore imageStore) {
         double totalCapacity = imageStoreStats.getCapacityBytes();
         double usedCapacity = imageStoreStats.getByteUsed();
         double threshold = getImageStoreCapacityThreshold();
-        String readableTotalCapacity = FileUtils.byteCountToDisplaySize((long) totalCapacity);
-        String readableUsedCapacity = FileUtils.byteCountToDisplaySize((long) usedCapacity);
+        String readableTotalCapacity = NumbersUtil.toReadableSize((long) totalCapacity);
+        String readableUsedCapacity = NumbersUtil.toReadableSize((long) usedCapacity);
 
-        logger.debug(String.format("Verifying image storage [%s]. Capacity: total=[%s], used=[%s], threshold=[%s%%].", imageStoreId, readableTotalCapacity, readableUsedCapacity, threshold * 100));
+        logger.debug(String.format("Verifying image storage [%s]. Capacity: total=[%s], used=[%s], threshold=[%.2f%%].", imageStoreId, readableTotalCapacity, readableUsedCapacity, threshold * 100));
```
Comment (DaanHoogland) [inline server/src/main/java/com/cloud/server/StatsCollector.java:1829]: any reason not to chenge to modern log format as well?

Comment (lucas-a-martins) [reply server/src/main/java/com/cloud/server/StatsCollector.java]: Changed in the latest commit

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Review by FelipeM525 — APPROVED** [review]
CLGTM!

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Comment by DaanHoogland** [comment]
@FelipeM525 @gpordeus , any testing done other than @lucas-a-martins ' own testing?

**Review by hsato03 — APPROVED** [review]
LGTM. Manually tested in a local lab.

**Pull request merged**